win32ss/user/user32/windows/cursoricon.c

```diff
@@ -1148,6 +1149,16 @@ BITMAP_LoadImageW(
         goto end;
     CopyMemory(pbmiCopy, pbmi, iBMISize);
 
+    /* Test if MSVC Resource Compiler used and if so, adjust pvBits */
+    HdrSize = *(char*)pbmi;
+    if (pbmiCopy->bmiHeader.biSizeImage + HdrSize + 12 == ResSize
+             && compr == BI_BITFIELDS && bpp == 32)
```

Comment:
Aren't these 12 bytes simply the bitfields, i.e. they would be present in any BI_BITFIELDS image?
It seems to essentially be a "palette" that tells you which bits in each pixel specify R vs G vs B, respectively: https://www.virtualdub.org/blog2/entry_177.html